### ISSUE TITLE
GLTF Mesh Normals Error

### ISSUE THREAD

**Issue by sebastianpetrovski** [cables-gl/cables_docs#714]
**Describe the bug**
There is a mesh that is rendering with an unwanted rectangle on it's surface if I plug in a Normal Map. Please see screenshot below.

![image](https://github.com/cables-gl/cables_docs/assets/74696962/933bce29-9afc-4ab7-a200-440b8f2e5de1)

**How To Reproduce**
In Cables.GL invited you @pandrr and @steam0r to collaborate on a mini testing patch I made for this issue. When you get a chance to look can you please let me know what you think?

I have tried many different combinations of settings in cables, different normal maps and inspected the mesh in Blender.

At first I thought it was a problem with the UV mapping or the Normal map, so I tried 3 different versions (in the patch) - but that's not it. Even just a blank colour for a Normal Map reproduces this rectangle in the same spot.

Also the mesh looks ok, it has all clean Tris and if I import the GLB to blender and apply the materials there it looks fine.

**Other things I've tried and annotated in the mini testing patch:**

If I toggle the Normals texture Op "Active" to off it changes it visibly but the rectangle remains.

If I change the "Calc Normals" option from "Never" to "Force Smooth" on the GltfScene Op it removes the rectangle. However, it visibly ruins the curvature in other areas of the mesh and you can see hard lines where there was normally a smooth curve.

Also If I disconnect the Normals texture the rectangle goes away.

**Platform**
This reproduces on Windows 11 and Android Chrome browsers. I haven't tested any others.

**Comment by steam0r** [comment]
please put a link to a simple patch here, that makes it easier/possible for others to help as well.

you may set it to "unlisted with editor": https://cables.gl/docs/faq/features/sharing/sharing

thanks!

**Comment by sebastianpetrovski** [comment]
Hey @steam0r - thanks for your reply :)

ok here it is: https://cables.gl/p/ArVKm9

**Comment by sebastianpetrovski** [comment]
I tried a few silly things today and discovered it seems the GLB file was corrupt.

I didn't think to re-process the mesh before because I batch exported the the offending GLB with 4 other GLBs and the others all worked fine in cables. 

In the end, the solution was to import the GLB from cables to blender, do nothing to the file and re-export it as GLB from Blender. I tried the re-export with and without compression (blender defaults) - both worked.
